lib/rutabaga/feature.rb

```diff
@@ -3,67 +3,22 @@
 
 module Rutabaga
   module Feature
-    def feature(feature_file = find_feature)
-
+    def feature(feature_file = nil)
+      RSpec.deprecate(
+        "Calling `feature` from an `it` block",
+        :message => "Calling `feature` from an `it` block " \
+                    "is deprecated.\nIt should now be called directly in the " \
+                    "`describe` block."
+      )
+
+      feature_file = Util.find_feature(feature_file || RSpec.current_example.description)
```

Comment:
Does it mean you can use the content of the `describe` argument as the file path? why it that not in the examples? it would be a even better solution than using `feature` in my opinion.
this kind of follow the same structure as the rspec rails view tests